substrate/client/network/src/request_responses.rs

```diff
@@ -746,7 +857,7 @@ impl NetworkBehaviour for RequestResponsesBehaviour {
 						// Received a response from a remote to one of our requests.
 						request_response::Event::Message {
 							peer,
-							message: Message::Response { request_id, response },
+							message: request_response::Message::Response { request_id, response },
```

Comment:
```suggestion
							message: Message::Response { request_id, response },
```

Reply:
We would need to completely remove `libp2p` from `request_response`. This task is very difficult because of the extensive use of `ToSwarm`.

Reply:
See the previous comment)